scripts/run_tests.sh

```diff
@@ -6,4 +6,5 @@ flake8 --max-complexity 6 src/caching_service
 flake8 src/test
 mypy --ignore-missing-imports src
 bandit -r src/caching_service
-python -m unittest discover src/test/caching_service
+coverage run --source=src/caching_service -m unittest discover src/test/caching_service
+coverage report
```

Comment:
Could you also add a ticket to convert to pytest?

Reply:
Do you mean just this line, or the entire test codebase, from unittest to pytest?

Reply:
These 2 lines. pytest is compatible with unittest

Reply:
converted to pytest

Reply:
It uses coverage to wrap the pytest run

Reply:
Yeah. It should just look something like this: https://github.com/kbase/cdm-task-service/?tab=readme-ov-file#running-tests

There's no need to wrap it in coverage

Reply:
Ok. Done

Reply:
Actually I just realized that there hasn't been a post from codecov so it seems that's not working yet...?

Reply:
I'm not sure; it seems like it cannot process the upload.

Reply:
Are you setting the right format for the coverage report? https://github.com/kbase/cdm-task-service/blob/main/.github/workflows/test.yml#L76

Reply:
yeah, that's what I thought. Let's see if it passes this time.